### PR TITLE
chore: combine both universal auth code flow suites

### PR DESCRIPTION
**Pull request by danielgent**
I keep noticing this. I thought I'd do this before adding more tests (we could always split these test suites up)

### CODE REVIEW

File: test/integration/login/code-flow.spec.ts
Comment (danielgent) [inline test/integration/login/code-flow.spec.ts:1]: this is the above test suite copy-pasted into here

**Review by markusahlstrand — COMMENTED** [review]
Hmm.. this is not a liquid-related test. Why move it?

**Comment by danielgent** [comment]
> Hmm.. this is not a liquid-related test. Why move it?

None are liquidjs tests now :smile: 

I'm not sure what the structure of these tests should be, but I'll revert this commit on the next PR

**Comment by danielgent** [comment]
It's too late Markus I've chained too many PRs up :smile: 

If I was using graphite this wouldn't be an issue as it would do it AFAIK :facepalm:   Sounds like I need to set that up

I'll revert this PR on a PR after merging the rest

**Pull request closed without merging**